### PR TITLE
Ft manage faqs

### PR DESCRIPTION
**Pull request by Maxcutex**
**What does this PR do?**
This PR allows admin to manage FAQs

**Description of Task to be completed?**

crud actions by admin for FAQs

**Any background context you want to provide?**

None

**What are the relevant pivotal tracker stories?**
159427828

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ class UserController < ApplicationController
   get '/logout' do
     if logged_in?
       session.clear
-      flash[:message] = 'You have logged out successfully!'
+      flash[:error] = 'You have logged out successfully!'
```
Comment (chadwalt) [inline app/controllers/users_controller.rb]: Do you really need this?

Comment (Maxcutex) [reply app/controllers/users_controller.rb]: Yea not really but just a convention it should be message not error !!! Thanks

File: app/controllers/faqs_controller.rb [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    end
+  end 
+
+  post '/postnew' do
+    begin
+      faqvals = { faq_title: params[:faq_title], faq_description: params[:faq_description]}
+      @faqs = FAQ.create(faqvals)
+      if @faqs.save
+        redirect to '/managefaqs'
+      else
```
Comment (chadwalt) [inline app/controllers/faqs_controller.rb]: You don't need the `else` remove it

File: app/controllers/faqs_controller.rb [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        data_table: false
+      }
+    end
+  end 
+
+  post '/postnew' do
+    begin
+      faqvals = { faq_title: params[:faq_title], faq_description: params[:faq_description]}
+      @faqs = FAQ.create(faqvals)
+      if @faqs.save
```
Comment (chadwalt) [inline app/controllers/faqs_controller.rb]: Add a blank line before the if statement to make it more readable

File: app/controllers/faqs_controller.rb [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+      }
+    end
+  end
+
+  post '/postedit' do
+    begin
+      faqvals = { faq_title: params[:faq_title], faq_description: params[:faq_description]}
+      @faqs = FAQ.where(id: params[:id]).first
+      @faqs.update(faqvals)
+      if @faqs.save
```
Comment (chadwalt) [inline app/controllers/faqs_controller.rb]: Add a blank line before the if statement

Comment (Maxcutex) [reply app/controllers/faqs_controller.rb]: done .. refactored to until

File: app/controllers/faqs_controller.rb (hunk 3 above)
Comment (Maxcutex) [reply app/controllers/faqs_controller.rb]: done .. refactored to until

**Pull request merged**